### DISCUSSION TITLE
how to get ALL text and images from a webpage ?

### DISCUSSION THREAD

**Discussion by sandys** [tusharojha/web_scraper#40] [Q&A] [answered]
hi,
I'm looking to get all the text and images from a webpage. I dont care about the formatting. Just that all the text & images is there. 

Is there anyway to do this ?

The thing i dont want to do is tie it to the structure of a particular webpage.  It should work on any page html

**Comment by tusharojha** [comment, marked as answer]
Hi Sandeep!
You do this easily using getElementAttribute method. Check the following example for images:

```dart
final images = webScraper.getElementAttribute("img", 'src');
images.forEach((link) => print(link));
```

**Reply by sandys** [reply]
@tusharojha thanks!
how would you do it for text ...since there is no one element that matches for text

**Reply by tusharojha** [reply]
Try this:

```dart
var elements = webScraper.getElementTitle('*');
```

**Reply by sandys** [reply]
@tusharojha i tried this, but doesnt work. 

```
import 'package:web_scraper/web_scraper.dart';

void main() async {
  final webScraper = WebScraper('https://twitter.com');
  if (await webScraper
      .loadWebPage('/washingtonpost/status/1357117167259561985')) {
    // final images = webScraper.getElementAttribute("img", 'src');
    // images.forEach((link) => print(link));
    var elements1 = webScraper.getElementTitle('*');
    elements1.forEach((link) => print(link));
  }
}

```
